compiler+runtime/bin/build-clang

```diff
@@ -40,31 +40,29 @@ srcdir="${PWD}"
 llvm_url="https://github.com/jank-lang/llvm-project.git"
 llvm_branch="main"
 
-function prepare()
-{
-  if [[ ! -d "${srcdir}/llvm" ]];
-  then
+function prepare() {
+  if [[ ! -d "${srcdir}/llvm" ]]; then
     git clone -b "${llvm_branch}" --shallow-submodules --single-branch "${llvm_url}" "${srcdir}"/llvm
   fi
 }
 
-function build()
-{
+function build() {
   mkdir -p "${srcdir}/llvm-build"
   cd "${srcdir}/llvm-build"
 
   cmake -DCMAKE_BUILD_TYPE=Release \
-        -DCMAKE_CXX_STANDARD=20 \
-        -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra" \
-        -DLLVM_TARGETS_TO_BUILD="host" \
-        -DLLVM_ENABLE_EH=OFF \
-        -DLLVM_ENABLE_RTTI=OFF \
-        -DLLVM_INCLUDE_BENCHMARKS=OFF \
-        -DLLVM_INCLUDE_EXAMPLES=OFF \
-        -DLLVM_INCLUDE_TESTS=OFF \
-        -DLLVM_ENABLE_ZSTD=OFF \
-        -G "Unix Makefiles" \
-        ../llvm/llvm
+    -DLLVM_ENABLE_RUNTIMES=all \
```

Comment:
Your change only affected this one line. Please don't re-format the whole file along with that one line change.